src/ContainerResolver.php

```diff
@@ -99,4 +131,19 @@ private static function resolveMvcContainer(): ContainerInterface
 
         return $serviceManager;
     }
+
+    /**
+     * @psalm-param non-empty-string $containerPath
+     */
+    private function resolveContainerFromPath(string $containerPath): ContainerInterface
+    {
+        if (! file_exists($containerPath)) {
+            throw new InvalidArgumentException('Provided path must be relative to the project root.');
```

Comment:
True, it can be an absolute path aswell.
Actually, this method is re-used by the original implementation which just used `include`.
So in 2/3 use-cases, the variable is relative, but yes - with the `container` option I do also allow absolute paths.

Maybe I should use different `resolve` methods to handle that.